docs/specs/git.yml

```diff
@@ -1,6 +1,6 @@
 ---
 # Show git contributors, author, updatedAt, enableContribution, editLink
-repo: https://github.com/docascode/docfx-test-dependencies#git-test
+repo: https://github.com/docascode/docfx-test-dependencies#git-test-5
```

Comment:
>git-test-5 [](start = 59, length = 10)

Why these test's branch is updated?
Suggest to create a new branch for new test or breaking changes, to avoid breaking the existing tests before PR merged.

Reply:
the test has changed, so you can treat this test is a new test, so i create another branch avoid to break old tests

---
In reply to: [202565048](https://github.com/dotnet/docfx/pull/3065#discussion_r202565048) [](ancestors = 202565048)

Reply:
got it. Seems we need to clean the test data repo for outdated test data later.

---
In reply to: [202565114](https://github.com/dotnet/docfx/pull/3065#discussion_r202565114) [](ancestors = 202565114,202565048)

Reply:
yes, will do it later.

---
In reply to: [202565498](https://github.com/dotnet/docfx/pull/3065#discussion_r202565498) [](ancestors = 202565498,202565114,202565048)